bin/docker/docker_setup.sh

```diff
@@ -0,0 +1,7 @@
+java -version
```

Comment:
I don't think this should be here.

Reply:
Oops let me delete it. Messed up while rebasing